models/vital_sign.go

```diff
@@ -1,5 +1,6 @@
 package models
 
 type VitalSign struct {
-	LabResult `bson:",inline"`
+	Entry            `bson:",inline"`
```

Comment:
Why this change? I made this a `LabResult`, because in HDS it inherits `LabResult`